openedx/core/djangoapps/credentials/migrations/0001_initial.py

```diff
@@ -1,6 +1,7 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
+from __future__ import absolute_import
```

Comment:
This import should be on the same line as the one above; I think these migration files have already been handled as part of INCR-215, so you should be able to revert your changes to these and then rebase from master.